src/release/breaking-changes/dispose.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Added missed `dispose()` for some disposable objects in Flutter
+description: >
+  'dispose()' may fail because of double disposal.
+---
+
+## Summary
+
+Missed 'dispose()' are added for some disposable objects. If some other code
+also invokes 'dispose()' for the objects, and the object is protected from double
+disposal, the second 'dispose()' will fail with error message:
+
+`Once you have called dispose() on a <class name>, it can no longer be used.` 
+
+## Background
+
+There is a rule that owner of an object should dispose it.
+
+This rule was broken in some places in Flutter Framework: owners were not disposing the disposable objects.
+The issue was fixed by adding `dispose()`. The added `dispose()` can cause failures in debug mode,
+if there is another `dispose()`, and the object is protected from double disposal.
+
+## Migration guide
+
+If you got error like `Once you have called dispose() on a <class name>, it can no longer be used.`,
+and the error is raised for your code, 
+update the code to call `dispose()' only in cases when your code created the object.
```

Comment:
**issue:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).
````suggestion
If the following error was raised for your code, 
update the code to call `dispose()' only in cases when your code created the object.

```nocode
Once you have called dispose() on a <class name>, it can no longer be used.
```
````

Reply:
Please fix the grammar:
If you encounter the following error, then remove
any additional calls in your code that dispose of the object:

```nocode
Once you have called dispose() on a <class name>, it can no longer be used.
```

Reply:
Shams, in most cases, the code should not be removed, but condition should be added. So, it should be fixed, not removed.